test/duct/env_test.clj

```diff
@@ -0,0 +1,27 @@
+(ns duct.env-test
+  (:require [clojure.test :refer :all]
+            [duct.core.env :as env]))
+
+(deftest coerce
+  (are [a b c] (= (env/coerce a b) c)
+    "a" 'Str "a"
+    12 'Str "12"
+    "123" 'Int 123
+    "true" 'Bool true
+    "TruE" 'Bool true
+    "t" 'Bool true
+    "" 'Bool false
+    "no" 'Bool false
+    nil 'Bool false)
+
+
+  (is (thrown? ArithmeticException (/ 1 0)))
```

Comment:
This test isn't testing anything in Duct. Accidentally included?

Reply:
Yes. Removed